src/journal/journald-server.c

```diff
@@ -677,6 +677,14 @@ static int server_archive_offline_user_journals(Server *s) {
 
                 TAKE_FD(fd); /* Donated to journal_file_open() */
 
+#if HAVE_GCRYPT
+                /* Write the final tag */
+                if (JOURNAL_HEADER_SEALED(f->header) && journal_file_writable(f)) {
+                        r = journal_file_append_tag(f);
+                        if (r < 0)
+                                log_warning_errno(r, "Failed to append tag when closing journal, ignoring: %m");
```

Comment:
should not log about LOG_DEBUG level

Reply:
This condition was previously logged as `log_error`, but I changed it now to be `log_debug` in the helper and with that in each occurence.